lib/ecstatic/showdir.js

```diff
@@ -110,6 +110,7 @@ module.exports = function (opts, stat) {
             '    <meta charset="utf-8">',
             '    <meta name="viewport" content="width=device-width">',
             '    <title>Index of ' + he.encode(pathname) +'</title>',
+            '    <style>@import url(https://fonts.googleapis.com/css?family=Open+Sans); * { margin:0; padding:0; -webkit-box-sizing:border-box; -moz-box-sizing:border-box; box-sizing:border-box; } html { min-height:100%; border:10px solid #ECEEF1; color:#61666c; font-weight:400; font-size:1em; font-family:"Open Sans", sans-serif; line-height:2em; } body { margin:0 auto; max-width:80%; padding:20px; -webkit-backface-visibility:hidden; } h1 { color:#2a2a2a; font-weight:400; margin-bottom:1em; } a { color:#61666c; text-decoration:none; } a:hover { color:#2a2a2a; } table { width:100%; table-layout:fixed; border-collapse:collapse; font-size:.925em; } tr:hover td { background:#f6f6f6; } td { padding:5px 0; outline:0; border:0; overflow:hidden; border-bottom:1px solid #edf1f5; vertical-align:middle; text-align:left; -webkit-transition:background 196ms ease; -moz-transition:background 196ms ease; -ms-transition:background 196ms ease; -o-transition:background 196ms ease; transition:background 196ms ease; } td a { display:block; } address { font-size:80%; text-align:left; font-style:normal; } address a { color:#f32236; text-decoration:none; } address a:hover { color:#f37c4c; }</style>',
```

Comment:
`font-family: "Open Sans",Verdana,Helvetica,sans-serif;` or something similar should get you close enough without having the dependency.